css/index.css

```
@@ -124,6 +124,30 @@ a:hover {
     margin-left: 0.25rem;
 }
 
+#projects ul {
+    margin-top: 2rem;
+    display: flex;
+    flex-wrap: wrap;
+    gap: 1rem;
+    max-width: 100vw;
+    justify-content: center;
+}
```

```suggestion
#projects ul {
    margin-top: 2rem;
    display: flex;
    flex-wrap: wrap;
    gap: 1rem;
    max-width: 100vw;
    justify-content: center;
    list-style: none;
}
```

Quick suggestion so that the dots before each project disappears :)